roles/rsyslog/roles/input_roles/files/defaults/main.yaml

```diff
@@ -68,7 +68,7 @@ rsyslog_conf_imfile_inputs:
       - comment: 'Log messages from log files'
         options: |-
           # FIXME: Need to normalize a log before passing it to elasticsearch output
-          {% if rsyslog_elasticsearch is defined and rsyslog_elasticsearch|length > 0 %}
+          {% if rsyslog_elasticsearch is defined and rsyslog_elasticsearch | length > 0 %}
```

Comment:
if `rsyslog_elasticsearch` is a `list`, `dict`, or `string`, you do not need to test the length.  https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html and in particular https://jinja.palletsprojects.com/en/2.11.x/templates/#if - you can just use `if rsyslog_elasticsearch is defined and rsyslog_elasticsearch` - this means `if rsyslog_elasticsearch is defined and rsyslog_elasticsearch is not empty`